RELEASE_NOTES.md

```diff
@@ -9,3 +9,5 @@
 - Adds a Formula Engine that can be used to evaluate formulas given component values.
 
 ## Bug Fixes
+
+- removes unwrap in partial equation evaluation
```

Comment:
You don't need to add this line.  It is only for bugs in previous releases that are fixed in this cycle.  When fixing bugs introduced in this release cycle or if this will be the first release, we don't need to mention fixes.